Makefile

```diff
@@ -7,7 +7,8 @@ GENCOMMIT = `git rev-list -1 HEAD`
 gen:
 	find gen/suites -maxdepth 1 -mindepth 1 -type d -print0 | xargs -I '{}' -n1 -0 bash -c 'dir="$$(basename {})" && echo "=== $${dir} ===" && cd {} && go run -ldflags "-X github.com/filecoin-project/test-vectors/gen/builders.GenscriptCommit=${GENCOMMIT}" . -o "../../../corpus/$${dir}"'
 
-
+compare-with-lotus-next:
+	pushd gen && go get github.com/filecoin-project/specs-actors@master && go get github.com/filecoin-project/lotus@a0c0d9c98aae && popd && cp -R corpus corpus-current && make gen && mv corpus corpus-new && ./diff.sh
```

Comment:
Hm... how about a parametrisable target that takes two arguments?

```shell script
$ make compare-corpus lotus next
$ make compare-corpus specs-actors master
```

* Argument 1: a string that's interpolated with `github.com/filecoin-project/${arg}`
* Argument 2: a string that's passed as-is as the argument after the `@` in go get.

Reply:
Maybe make all this magic part of the diff script so that: (a) we can easily invoke it locally, (b) it cleans up after itself?

Reply:
@raulk I liked that idea and I implemented it, but at the moment we can't upgrade specs-actors to master as our tool doesn't compile anymore, so I am not convinced we can just update lotus or specs-actors and expect for the test vectors to continue to be generated.